src/components/trade/PriceSuggestions/PriceSuggestionItem.tsx

```diff
@@ -26,20 +27,34 @@ interface FormattedPrices {
   inversePriceValue: string
 }
 
-const LOW_PRICE_FLOOR = new BigNumber('0.0001')
-
-export function formatPriceToPrecision(price: BigNumber, useThreshold = false): string {
-  return price.gt(LOW_PRICE_FLOOR) || !useThreshold
-    ? price.toFixed(PRICE_ESTIMATION_PRECISION)
-    : '< ' + LOW_PRICE_FLOOR.toString()
+interface GetPriceFormatted {
+  price: BigNumber | null
+  isPriceInverted: boolean
+  baseTokenDecimals: number
+  quoteTokenDecimals: number
 }
 
-function getPriceFormatted(price: BigNumber | null, isPriceInverted: boolean): FormattedPrices {
+function getPriceFormatted({
+  price,
+  isPriceInverted,
+  baseTokenDecimals,
+  quoteTokenDecimals,
+}: GetPriceFormatted): FormattedPrices {
   if (price) {
-    const inversePriceLabel = formatPriceToPrecision(invertPrice(price), true)
-    const priceLabel = formatPriceToPrecision(price, true)
-    const inversePriceValue = invertPrice(price).toString(10)
-    const priceValue = price.toString(10)
+    const inversePriceLabel = formatPriceWithFloor(invertPrice(price))
+    const priceLabel = formatPriceWithFloor(price)
+
+    // Use long precision form for accuracy
+    // Use BigNumber's decimalPlaces(<# of decimal places to show>, ROUNDING_MODE)
+    // Rounds away 0's
+
+    // ROUNDING_MODE [1] => Rounds towards zero
```

Comment:
this comment won't be needed when u remove the magic numbers

Reply:
yup